### PR TITLE
data:Malavi-Pande

### PR DESCRIPTION
**Pull request by Malavi1**

### CODE REVIEW

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey,

Please add your profile in `data` which is at the root of the project. 

Also, fix your JSON profile file naming. It should be `{your GitHub username}.json` ->  `Malavi1.json`. Otherwise, some features will not be functional in the future.

If you need any help feel free to ask.

File: data/Malavi1.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      "color": "grey",
+      "description": "Participated in my first ever Hacktoberfest in this path I make a very good friends",
+      "url": "https://github.com/Malavi1"
+    },
+    {
+      "title": "Started Bloging",
+      "date": "January 2023",
+      "icon": "SiHashnode",
+      "color": "blue",
+      "description": "It helping me to document my Journey comprehensively through blogs,I am one of the top 10 participants of #Devretro2022"
```
Comment (Panquesito7) [inline data/Malavi1.json:48]: Missing comma
```suggestion
      "description": "It helping me to document my Journey comprehensively through blogs,I am one of the top 10 participants of #Devretro2022",
```

File: data/Malavi1.json [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      "url": "https://hashnode.com/@Malavi"
+    },
+    {
+      "title": "Started Contributing",
+      "date": "January 2023",
+      "icon": "SiGithub",
+      "color": "blue",
+      "description": "I feel fortunate that I get to know about the open source and started contributing"
```
Comment (Panquesito7) [inline data/Malavi1.json:56]: Same here.
```suggestion
      "description": "I feel fortunate that I get to know about the open source and started contributing",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good to merge. Thanks for your Contribution! 🎉 

<p align="center"></p>

**Pull request merged**